.meteor/packages

```diff
@@ -12,7 +12,6 @@ reactive-var@1.0.10            # Reactive variable for tracker
 jquery@1.11.9                  # Helpful client-side library
```

Comment:
I still see the insecure package in this file.  This should be removed at some point.  The package says it is good for prototyping, but I find it annoying because you might not find access issues until after you remove the package.